### ISSUE TITLE
180 long crossing section can be improved

### ISSUE THREAD

**Issue by SrNetoChan** [qgis/QGIS-Documentation#1911]
The following subsection can be detached from the PostGIS section and improved to include raster files and other solutions to the problem (using GDAL/OGR, virtual layers) 

http://docs.qgis.org/2.18/en/docs/user_manual/working_with_vector/supported_data.html#vector-layers-crossing-180-degrees-longitude

**Comment by DelazJ** [comment]
Yeah, another bits of changes required in the so called "supported format" chapter (#1088).
But, where could that go? what kind of description could that be linked to?

**Comment by SrNetoChan** [comment]
@DelazJ If we have a generic supported data formats (one of your proposals I think) we could have this in a separate subsection (corner cases?). It's a problem that crosses both the raster and vector format.

Same goes for the date time storing problems. something in the end of the chapter.
